src/natcap/invest/recreation/recmodel_client.py

```diff
@@ -1502,6 +1505,33 @@ def error_handler(err_level, err_no, err_msg):
             "projection of the base vector")
 
 
+def _validate_predictor_types(table_path):
+    """Validate the type values in a predictor table.
+
+    Args:
+        table_path (string): path to a csv table that has at least
+            the field 'type' 
+
+    Returns:
+        None
+
+    Raises:
+        ValueError if any value in the ``type`` column does not match a valid
+        type, ignoring leading/trailing whitespace.
+    """
+    df = utils.read_csv_to_dataframe(table_path, to_lower=True)
+    type_list = df['type'].tolist()
+    valid_types = ['raster_mean', 'raster_sum', 'point_count', 
+                   'point_nearest_distance', 'line_intersect_length',
+                   'polygon_area_coverage', 'polygon_percent_coverage']
+    for type_value in type_list:
+        # ignore leading/trailing whitespace because it will be removed
+        # when the type values are used
+        if type_value.strip() not in valid_types:
+            raise ValueError(f'The table contains an invalid type value: \
```

Comment:
The backlash `\` for strings over multiple lines works here, but inside parens you can also extend strings like:
```python
raise ValueError(
    'The table contains an invalid type value:'
    f' {type_value}. The allowed types are: {", ".join(valid_types)}')
```
I end up doing this a lot and feel that it is a little more clean than the backlash...